sandbox/src/routes/generatePatientLetter.js

```diff
@@ -0,0 +1,39 @@
+const mockResponseProvider = require('../services/mockResponseProvider')
+const businessFunctionValidator = require('../services/businessFunctionValidator')
+
+module.exports = [
+  /**
+   * Sandbox implementation for generatePatientLetter endpoint
+   */
+  {
+    method: 'POST',
+    path: '/STU3/v1/ReferralRequest/{ubrn}/$ers.generatePatientLetter',
+    handler: (request, h) => {
+
+      const allowedBusinessFunctions = ["REFERRING_CLINICIAN", "REFERRING_CLINICIAN_ADMIN"]
+
```

Comment:
This is true at the moment, but will not be once the changes I am currently working on are committed.

This raises the issue of what proceedures are we putting in place for making sure that any functional changes are reflected in this documentation ( not an issue for this review - just thinking out loud really)

Reply:
I hope that once this becomes source of truth it will get updated along with each ticket we complete